templates/taxi/car_detail.html

```diff
@@ -19,4 +19,13 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if user not in car.drivers.all %}
+  <a href="{% url 'taxi:car-driver-assign' pk=car.id %}" class="btn btn-primary">
+    Assign me to this car
+  </a>
+  {% else %}
+  <a href="{% url 'taxi:car-driver-assign' pk=car.id %}" class="btn btn-danger">
+    Delete me from this car
+  </a>
+  {% endif %}
```

Comment:
Ensure that the `assign_user_to_car` view function is correctly configured in the URL patterns and that it handles the logic for assigning and removing drivers from cars. This will ensure that the buttons for 'Assign me to this car' and 'Delete me from this car' function as expected.